DownloadFile.ps1

```diff
@@ -9,12 +9,18 @@ $ErrorActionPreference = 'Stop'
 
 $latestRelease = Invoke-RestMethod -Uri "https://api.github.com/repos/StefanMaron/BusinessCentral.LinterCop/releases"
 
-if ($prerelease -eq "false") {
+$prerelease = $prerelease.ToString().ToLower()
```

Comment:
doesn't this lead to an error if `$prerelease` is `$null`?

shouldn't we check at first if the value is null and and only if the value isn't null perform conversions to String or other checks?

Reply:
You're right, this will indeed throw an error. I've changed this to handle a possible $null input value.

Thank you for taking the time to review this!